src/Moryx.Runtime.Endpoints/Modules/Endpoint/ModulesController.cs

```diff
@@ -156,7 +156,7 @@ public ActionResult ConfirmWarning([FromRoute] string moduleName)
 
             var notifications = module.Notifications.ToArray();
             foreach (var notification in notifications)
-                notification.Confirm();
+                module.AcknowledgeNotification(notification);
```

Comment:
Should we remove all the notification in one step, or they should be acknowledged one by one?

Reply:
In that case it could help, but it would require another method in the module API.